www_src/components/tabs/tabs.jsx

```diff
@@ -31,7 +31,7 @@ var Tabs = React.createClass({
             'tab-body': true,
             'selected': this.state.selectedIndex === i
           });
-          return (<div className={className}>{tab.body}</div>);
+          return (<div key={i} className={className}>{tab.body}</div>);
```

Comment:
Hmm... if the icon is tied to the tab, that's better than `i`, probably. That way even if the tabs array is reordered, React will still correctly identify each tab as "the one it should be".